lib/create-server.js

```diff
@@ -22,9 +22,15 @@ function createServer (argv, app) {
   app.use(mount, ldpApp)
   debug.settings('Base URL (--mount): ' + mount)
 
+  if (argv.idp) {
+    console.error('The idp configuration option has been renamed to multiuser.')
```

Comment:
Nitpick: may as well use `console.warn` so as not to print out a stack trace.

Reply:
Good idea! This was copied from a similar warning elsewhere; will adjust that too.

Reply:
Done in 9e1b59fa631008a98c06b1f0689711fbfe775793.